storage/innobase/srv/srv0srv.cc

```diff
@@ -1713,3 +1713,22 @@ void srv_purge_shutdown()
     srv_shutdown_purge_tasks();
   }
 }
+
+uint32_t srv_cleanout_threads = 8;
+my_bool srv_use_scn = FALSE;
```

Comment:
The default values should already set via the data structure in `ha_innodb.cc`. We can let these variables live in the BSS. I’d like the variables for any new parameters to be named in the same way as the configuration parameter, e.g., `innodb_use_scn`.

Reply:
OK